### PR TITLE
Use init generated files, clean up repo

### PR DESCRIPTION
**Pull request by andrewvc**
Replaces the contents of the todo folder in this repo with the output of the `init` command from https://github.com/elastic/synthetics/pull/525

This also adds examples using the `request` object filling a gap identified in the documentation of that feature (CC @colleenmcginnis ) 

It also removes the `e-commerce` demo temporarily till we have steady hosting for it. This blocks #25 as a result.

Finally, this adds inline examples, and removes the `heartbeat` folder, since as of 8.4 we want to discourage running browser monitors directly with heartbeat.

DO NOT MERGE THIS this until 8.4 is publicly available and beta 32 of the synthetics CLI is out

### CODE REVIEW

File: advanced-examples/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "push": "npx @elastic/synthetics push --auth $SYNTHETICS_API_KEY"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@elastic/synthetics": "^1.0.0-beta.31"
+  },
+  "devDependencies": {
+    "@types/node": "^18.7.11"
```
Comment (vigneshshanmugam) [inline advanced-examples/package.json:17]: Did the init command added this types by default and also the lock files?

**Review by vigneshshanmugam — COMMENTED** [review]
Should be good to do once 
1. we change the selectors to Locators - https://github.com/elastic/synthetics/commit/e69335673474c0acb09e4fdfc1bce7e55f59bb90
2. Update the version to `1.0.0-beta.32`
3. Fix the lock files

File: advanced-examples/package.json (hunk 1 above)
Comment (andrewvc) [reply advanced-examples/package.json:17]: No, I added it. We should add that to the init command, otherwise vs code's completion was broken

Comment (andrewvc) [reply advanced-examples/package.json:17]: To be more specific it's required by our use of `Buffer`

**Review by vigneshshanmugam — APPROVED** [review]
LGTM pending these changes

1. Change the version to `beta-32` - https://github.com/elastic/synthetics-demo/blob/0efcf05bb46406a4b3dcb340ca054a7c406d165f/advanced-examples/package.json#L14 and https://github.com/elastic/synthetics-demo/blob/0efcf05bb46406a4b3dcb340ca054a7c406d165f/todos/package.json#L14
2. Remove `.synthetics` duplicate - https://github.com/elastic/synthetics-demo/blob/0efcf05bb46406a4b3dcb340ca054a7c406d165f/todos/.gitignore#L3

**Comment by shahzad31** [comment]
Skipping POST FF Testing !!

**Pull request merged**